### PR TITLE
Grammar Fixes

### PR DESCRIPTION
**Pull request by daggy1234**
Found typo's in the array rustdoc. Pr'ed a fix!

### CODE REVIEW

**Review by henryboisdequin — APPROVED** [review]

File: library/core/src/array/mod.rs [hunk 1]
```diff
@@ -382,7 +382,7 @@ impl<T: Ord, const N: usize> Ord for [T; N] {
 
 // The Default impls cannot be done with const generics because `[T; 0]` doesn't
 // require Default to be implemented, and having different impl blocks for
-// different numbers isn't supported yet.
+// different numbers aren't supported yet.
```
Comment (JohnTitor) [inline library/core/src/array/mod.rs:385]:
```suggestion
// different numbers isn't supported yet.
```

This should be `isn't`: `having ~ isn't supported yet.`

**Review by JohnTitor — CHANGES_REQUESTED** [review]

File: library/core/src/array/mod.rs (hunk 1 above)
Comment (Daggy1234) [reply library/core/src/array/mod.rs:385]: Oh whoops my bad

**Review by JohnTitor — APPROVED** [review]
👍

**Pull request merged**